### PR TITLE
Refactor members adding/ removing.

### PR DESCRIPTION
**Pull request by ben-kaufman**
Small refactoring.

### CODE REVIEW

**Comment by orenyodfat** [comment]
is this related to specific issue ?

**Comment by ben-kaufman** [comment]
Not exactly. I did it when investigating this issue #208 but it's not exactly related (probably).

File: src/domain/member.ts [hunk 1]
```diff
@@ -50,7 +51,7 @@ export function updateMemberReputationWithValue(
 ): void {
   let member = getMember(address, daoAddress);
   member.reputation = value;
-  if (equals(member.reputation as BigInt, BigInt.fromI32(0))) {
+  if (equals(member.reputation, BigInt.fromI32(0)) && equals(member.tokens, BigInt.fromI32(0))) {
```
Comment (orenyodfat) [inline src/domain/member.ts]: does and why token holders are dao members ?

Comment (ben-kaufman) [reply src/domain/member.ts]: Yes and because that's what Alchemy needed

Comment (orenyodfat) [reply src/domain/member.ts]: plus @jellegerbrandy for clarification here.

Comment (jellegerbrandy) [reply src/domain/member.ts]: No, I think it is clearer if we members collection only contanis the reputation holders. (So I'd prefer not to merge this change)

**Pull request closed without merging**